robot/rospackages/src/task_handler/scripts/task_handler_client.py

```diff
@@ -40,24 +50,40 @@ def usage():
     """
     Return string showcasing proper usage of this client script
     """
-    help_msg = """USAGE:\nrosrun task_handler task_handler_client.py [task] [status] optional:[args]
-                  \nValid task options: ['arm_listener', 'arm_listener', 'science_listener', 'camera_stream']
-                  \nValid status options: [0, 1, 2]
-                  \nValid camera stream args: ['/dev/ttyFrontCam', '/dev/ttyRearCam', '/dev/ttyArmScienceCam']"""
+    global ports
+
+    help_msg = "USAGE:\nrosrun task_handler task_handler_client.py [task] [status] optional:[args]"
+    help_msg += "\nValid task options: ['arm_listener', 'arm_listener', 'science_listener', 'camera_stream']"
```

Comment:
why is the `arm_listener` valid twice lol?

You should probably make a string array of valid args and loop it here and reuse it when checking the args so its synced with the help

Reply:
like I mean use the array known_tasks

Reply:
yeah it's a typo it one of them should be `rover_listener`
I'll construct the string by using the `known_tasks` array

Reply:
Fixed